gym-unity/README.md

```diff
@@ -31,7 +31,7 @@ from the root of the project repository use:
 ```python
 from gym_unity.envs import UnityToGymWrapper
 
-env = UnityToGymWrapper(unity_environment, uint8_visual, allow_multiple_obs)
+env = UnityToGymWrapper(unity_environment, uint8_visual, flatten_branched, allow_multiple_obs)
```

Comment:
```suggestion
env = UnityToGymWrapper(unity_environment, uint8_visual, flatten_branches, allow_multiple_obs)
```

Reply:
I saw that both the description below and the code are using `flatten_branched`. Where did you see `flatten_branches`?

Reply:
Oh, sorry, I thought that was a typo, but I guess that's what the actual parameter is called.